sdks/js/packages/sdk-demo/src/config/frontier.ts

```diff
@@ -1,5 +1,12 @@
-const config = {
-  endpoint: '/api'
+import { FrontierClientOptions } from '../../../core/react/dist';
+
+const config: FrontierClientOptions = {
+  endpoint: '/api',
+  billing: {
+    basePlan: {
+      title: 'Standard Plan'
```

Comment:
Do we not need to specify price as 0?

Reply:
The fallback in the code shows 0 prices instead of `NaN.`
You can pass the price; it will override that 0.